tests/run-make/emit-to-stdout/rmake.rs

```diff
@@ -0,0 +1,60 @@
+// Test that emitting to stdout works
```

Comment:
Suggestion: (not reflowed, also please double check if this comment is accurate)

```suggestion
//! If `-o -` or `--emit KIND=-` is provided, output should be written
//! to stdout instead. Binary output (`obj`, `llvm-bc`, `link` and
//! `metadata`) being written this way will result in an error if
//! stdout is a tty. Multiple output types going to stdout will
//! trigger an error too, as they will all be mixed together.
//!
//! See <https://github.com/rust-lang/rust/pull/111626>.
```